openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -484,6 +484,7 @@
     "primary-key": "Primary Key",
     "private-key": "PrivateKey",
     "private-key-id": "Private Key ID",
+    "process-pii-sensitive-column": "Process Pii Sensitive Column",
```

Comment:
Should we rename this to "Auto tag PII Sensitive Columns"

Reply:
yes let's feature this as Auto PII. Also I think it should be `PII` instead of `Pii`? 🤔